src/com/itacademy/eshop/simulations/CustomerSimulation.java

```diff
@@ -29,22 +34,45 @@ private void addProductsToShoppingCart() {
          * User browses the shop and adds 4 products to the shopping cart.
          * One of them should be a book and other should be a shirt.
          */
-        throw new UnsupportedOperationException("addProductsToShoppingCart() is not implemented yet");
+//        throw new UnsupportedOperationException("addProductsToShoppingCart() is not implemented yet");
+
+        shoppingCart.addProduct(new Product("shirt", 15.90, Category.CLOTHING));
+        shoppingCart.addProduct(new Product("stories", 15.90, Category.BOOKS));
+        shoppingCart.addProduct(new Product("phone", 600.90, Category.ELECTRONICS));
+        shoppingCart.addProduct(new Product("lego", 35.90, Category.TOYS));
     }
 
     private void removeProductsFromShoppingCart() {
         /**
          * User removes shirt from the shopping cart.
          */
-        throw new UnsupportedOperationException("removeProductsFromShoppingCart() is not implemented yet");
+//        throw new UnsupportedOperationException("removeProductsFromShoppingCart() is not implemented yet");
+
+        shoppingCart.removeProduct(new Product("shirt", 15.90, Category.CLOTHING));
     }
 
-private void leaveReviewsAndRatingsForProducts() {
+    private void leaveReviewsAndRatingsForProducts() {
         /**
          * User leaves 3 reviews and ratings for the products in the shopping cart.
          */
-        throw new UnsupportedOperationException("leaveReviewsAndRatingsForProducts() is not implemented yet");
-    }
+//        throw new UnsupportedOperationException("leaveReviewsAndRatingsForProducts() is not implemented yet");
+
+        ArrayList<Product> product = shoppingCart.getProducts();
```

Comment:
Product is totally valid variable name. But here we have a collection of objects. Name should reflect that and be in plural form like `products`